UML editor/src/main/java/codecain/GraphicalInterface.java

```diff
@@ -0,0 +1,324 @@
+package codecain;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+
+public class GraphicalInterface extends JFrame {
+    private JPanel canvas;
+    private JPanel controlsPanel;
+    private JPanel classesPanel;
+
+    public GraphicalInterface() {
+        setTitle("Class Diagram Editor");
+        setSize(800, 400);
```

Comment:
Would like the starting window height to be taller so you can see all buttons on startup instead of having to resize the window once launched.